Assets/Scripts/Interactables/Pickup.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+#nullable enable
```

Comment:
`#nullable enable` allows us to create Nullable types in C#
i.e. `private Transform? mouthAttachmentPoint` creates new attribute that the editor will use to remind us if we don't  check for the null case.